hudi-utilities/src/main/java/org/apache/hudi/utilities/deltastreamer/HoodieDeltaStreamer.java

```diff
@@ -369,6 +369,15 @@ private boolean onDeltaSyncShutdown(boolean error) {
     @Parameter(names = {"--bootstrap-index-class"}, description = "subclass of BootstrapIndex")
     public String bootstrapIndexClass = HFileBootstrapIndex.class.getName();
 
+    @Parameter(names = {"--retry-on-source-failures"}, description = "Retry on any source failures")
```

Comment:
Possible to exercise these configs in one of the deltastreamer tests?

Reply:
I also wanted to add one, but tests will involve longer running times since we do explicit thread.sleep in source code. So, resorted to manual testing for now.